<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_menu</class>
 <widget class="QDialog" name="choose_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="choose_menu_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Gamemode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_menu_ai_choose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Easy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Impossible</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="choose_menu_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_menu_vs_player">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Versus Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_menu_vs_ai">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Versus AI</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>choose_menu_vs_player</tabstop>
  <tabstop>choose_menu_vs_ai</tabstop>
  <tabstop>choose_menu_ai_choose</tabstop>
  <tabstop>choose_menu_back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
